src/test/java/nl/dtl/fairmetadata4j/utils/MetadataUtilsTest.java

```diff
@@ -385,6 +539,27 @@ public void testGetDataRecordNoRMLURI() throws Exception {
     
     
     /**
+     * Test missing rdf statement but the mandatory check is disabled,
+     * so this test is excepted to pass
+     * 
+     * @throws Exception 
+     */
+    @Test
+    public void testGetDataRecordNoRMLURIWithFlag() throws Exception {        
+        System.out.println("Test : Missing rml uri for  datarecord metadata");
+        DataRecordMetadataParser parser = new DataRecordMetadataParser();
+        List<Statement> stmts = ExampleFilesUtils.getFileContentAsStatements(
+                ExampleFilesUtils.DATARECORD_METADATA_FILE, 
+                        ExampleFilesUtils.DATARECORD_URI.toString());
+        IRI drURI = ExampleFilesUtils.DATARECORD_URI;
+        DataRecordMetadata metadata = parser.parse(stmts , drURI);  
+        metadata.setRmlURI(null);
+        List<Statement> excepted = MetadataUtils.getStatements(metadata);
+        assertNotNull(excepted);
+    }
+    
+    
+    /**
```

Comment:
As a generic comment on the tests added to this class: there is a lot of repetition in each test setup. Maybe you can refactor it into a private convenience method, I can easily see about 6 lines saved for each method. With the methods added in this commit that's easily 40 lines saved.